### ISSUE TITLE
[deal 9.0.0] tests/stokes_residual.prm

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#1989]
on deal.II master stokes_residual.prm fails with
```
An error occurred in line <161> of file <../source/lac/solver_control.cc> in function
    const std::vector<double>& dealii::SolverControl::get_history_data() const
The violated condition was: 
    history_data.size() > 0
Additional information: 
    The SolverControl object was asked for the solver history data, but there is no data. Possibly you requested the data before the solver was run.
```

**Comment by gassmoeller** [comment]
This should be solved by #1988

**Comment by gassmoeller** [comment]
Reopening, because now stokes_residual_cheap fails.

**Comment by gassmoeller** [comment]
Fixed by #1993.
